### DISCUSSION TITLE
Is there advanced usage of :equal ?

### DISCUSSION THREAD

**Discussion by ipp2020** [AutomaApp/automa#1402] [Q&A]
Q: To solve the selection box selection element 's question.  I want to locate the 2) AA2
1)
<div el-scrollbar__view el-select-dropdown__list>
<li><span>AA1</span></li>
<li><span>AA2</span></li>
<div/>
2)
<div el-scrollbar__view el-select-dropdown__list>
<li><span>AA1</span></li>
<li><span>AA2</span></li>
<div/>

thanks very much!!!

**Comment by Delusionnnn** [comment]
Sorry this is a poorly written question here and I dont understand what you are referring to. What block are you using? Click element? Conditions? Are you just referring to the element selector? What is your goal in more plainly written english? I will try to help.

**Comment by dingzz3** [comment]
Review the elements and determine the tag name of AA2 (assuming it is tag a), then locate as      a: equal ('AA2 ')
